### PR TITLE
BLUE

### PR DESCRIPTION
**Pull request by anhnt170489**
In this PR: 
- I add BertForMultiLabelClassification, RobertaForTokenClassification, RobertaForMultiLabelClassification. 
- I add examples for Finetuning the BERT, RoBERTa models for tasks on BLUE (https://github.com/ncbi-nlp/BLUE_Benchmark). BLUE (Biomedical Language Understanding Evaluation) is similar to GLUE, but for Biomedical data. The "run_blue", "utils_blue" are customized from "run_glue", "utils_glue", but more sufficient, because it contains not only sequence classification, but also token classification, multi-label classification. People may also have more options for examples of fine-tuning BERT/RoBERTa. 
- I also add test function to test_examples as well as test data (HOC)

### CODE REVIEW

**Comment by thomwolf** [comment]
@LysandreJik @julien-c one of you want to give a look?

**Comment by anhnt170489** [comment]
@thomwolf : Hi, all checks have passed. Please review it again ;).

File: pytorch_transformers/modeling_roberta.py [hunk 1]
```diff
@@ -357,3 +357,142 @@ def forward(self, features, **kwargs):
...
+            list of ``torch.FloatTensor`` (one for the output of each layer + the output of the embeddings)
+            of shape ``(batch_size, sequence_length, hidden_size)``:
+            Hidden-states of the model at the output of each layer plus the initial embedding outputs.
+        **attentions**: (`optional`, returned when ``config.output_attentions=True``)
+            list of ``torch.FloatTensor`` (one for each layer) of shape ``(batch_size, num_heads, sequence_length, sequence_length)``:
+            Attentions weights after the attention softmax, used to compute the weighted average in the self-attention heads.
+
+    Examples::
+
+        tokenizer = RoertaTokenizer.from_pretrained('roberta-base')
```
Comment (dennlinger) [inline pytorch_transformers/modeling_roberta.py]: This should probably be renamed to `RobertaTokenizer` (typo).

File: pytorch_transformers/modeling_roberta.py [hunk 2]
```diff
@@ -357,3 +357,142 @@ def forward(self, features, **kwargs):
...
```
Comment (dennlinger) [inline pytorch_transformers/modeling_roberta.py]: Similar to the other highlight, his should probably be renamed to `RobertaTokenizer` (typo).

**Review by dennlinger — CHANGES_REQUESTED** [review]
Minor typos that make it hard to spot incosistencies later.

**Comment by anhnt170489** [comment]
This PR is out of date. 
I updated a new one here (https://github.com/huggingface/transformers/pull/1440).
